### PR TITLE
Hylics 2: Update option docstrings

### PR DESCRIPTION
**Pull request by TRPG0**

### CODE REVIEW

**Review by Exempt-Medic — APPROVED** [review]
They display well on WebHost

File: worlds/hylics2/Options.py [hunk 1]
```diff
@@ -1,25 +1,38 @@
...
+    
+    Note that enabling this can significantly increase both the difficulty and length of a run.
+    """
     display_name = "Shuffle Party Members"
 
+
 class GestureShuffle(Choice):
-    """Choose where gestures will appear in the item pool."""
+    """
+    Choose where gestures will appear in the item pool.
```
Comment (ScipioWright) [inline worlds/hylics2/Options.py:16]: This is such a tiny nitpick, but sometimes the option says "the pool" and sometimes it says "the item pool", so you may want to fix that up, or you can leave it alone, up to you.

File: worlds/hylics2/Options.py [hunk 2]
```diff
@@ -35,14 +48,23 @@ def get_option_name(cls, value: int) -> str:
...
+
 class Hylics2DeathLink(DeathLink):
-    """When you die, everyone dies. The reverse is also true.
+    """
+    When you die, everyone dies. The reverse is also true.
+    
     Note that this also includes death by using the PERISH gesture.
-    Can be toggled via in-game console command "/deathlink"."""
+    
+    Can be toggled via in-game console command "/deathlink".
```
Comment (ScipioWright) [inline worlds/hylics2/Options.py:65]: I don't know for sure if the tooltip supports `stuff like this`, but maybe you could try surrounding /deathlink with ` and see what it looks like on webhost? I'd check but I'm not at my home computer atm

Comment (Exempt-Medic) [reply worlds/hylics2/Options.py:65]: It seems like that doesn't work, unfortunately.

**Review by LegendaryLinux — APPROVED** [review]

**Pull request merged**